lnbits/nodes/cln.py

```diff
@@ -310,10 +310,10 @@ async def get_invoices(
         return Page(
             data=[
                 NodeInvoice(
-                    bolt11=invoice["bolt11"],
-                    amount=invoice["amount_msat"],
+                    bolt11=invoice.get("bolt11") if invoice.get("bolt11") != None else invoice.get("bolt12"),
+                    amount=invoice.get("amount_msat") if invoice.get("amount_msat") != None else invoice.get("amount_received_msat"),
```

Comment:
This is more idiomatic and captures better what we want to do (if value is set, we want to use it, otherwise throw an exception, we do not want to silently pass `None`).

```suggestion
                    bolt11=invoice["bolt11"] if "bolt11" in invoice else invoice["bolt12"],
                    amount=invoice["amount_msat"] if "amount_msat" in invoice else invoice["amount_received_msat"],
```

Reply:
@kallerosenbaum can you please test whether this suggested code works for you?

Reply:
> This is more idiomatic and captures better what we want to do (if value is set, we want to use it, otherwise throw an exception, we do not want to silently pass `None` because neither `bolt11` nor `amount` are optional fields).

these changes are not required as pydantic will throw an exception if we pass `None` to either of these fields. We can actually simplify it to
```python
bolt11=invoice.get("bolt11") or invoice.get("bolt12"),
```